### PR TITLE
Add ptp feature using kustomize

### PR DESCRIPTION
**Pull request by SchSeba**
This PR add the ptp feature using kustomize.

Signed-off-by: Sebastian Sch <sebassch@gmail.com>

---
This change is [Reviewable](https://reviewable.io/reviews/openshift-kni/performance-addon-operators/26)

### CODE REVIEW

File: cluster-setup/ci-cluster/feature_ptp/kustomization.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+bases:
+  - ../../base/feature_ptp
+patches:
+  - subscription.yaml
```
Comment (fedepaol) [inline cluster-setup/ci-cluster/feature_ptp/kustomization.yaml]: Why this patch? The target should be 4.4

**Comment by fedepaol** [comment]
Also, if the ptp operator requires some configuration (such as node labelling, iirc) this setup should be here.

**Comment by SchSeba** [comment]
Hi @fedepaol 

> Also, if the ptp operator requires some configuration (such as node labelling, iirc) this setup should be here.

I will like to do the label and everything related to that in the functional test itself it will give us better control and leave the deploy to using only kustomize

**Comment by fedepaol** [comment]
> Hi @fedepaol
> 
> > Also, if the ptp operator requires some configuration (such as node labelling, iirc) this setup should be here.
> 
> I will like to do the label and everything related to that in the functional test itself it will give us better control and leave the deploy to using only kustomize

Sounds good to me.

File: cluster-setup/ci-cluster/feature_ptp/kustomization.yaml (hunk 1 above)
Comment (SchSeba) [reply cluster-setup/ci-cluster/feature_ptp/kustomization.yaml]: here we use the test source.

Production: `redhat-operators`
Test: `opsrctest` the should point to the ART source catalog

**Comment by slintes** [comment]
with the new cnf deploy repo being available, I guess we do not want to add more features to this repo, but only test the operator itself?
@davidvossel ?

**Comment by slintes** [comment]
just found the confirmation of my assumption over here: https://github.com/openshift-kni/performance-addon-operators/pull/28#issuecomment-572595831

/close

**Pull request closed without merging**